.travis/custom_check_pull_request.py

```diff
@@ -66,7 +66,10 @@ def check_changelog_record(issue):
 
 def check_issue_exists(issue):
     response = requests.head(JIRA_URL.format(issue=issue))
-    if not response.ok:
+    if response.status_code == 404:
```

Comment:
Does jira ever respond with any other 4xx codes?

Reply:
@newswangerd if issue doesn't exist it responds with 404, as we only want to check if the issue exists any other code is valid, like 401 (unauthorized) or 200 which will return only if user is authenticated.

Jira can respond with server error (50x) but in that case do we want to block the CI?